Single use statement missing when generating libhandy-rs

I updated libhandy-rs to current gtk-rs master. Everything seems to work but I had to add a single `use gtk::Widget;` to the auto-generated code.

My branch is here https://gitlab.gnome.org/sophie-h/libhandy-rs/-/tree/gtk-rs-0.14

```rust
$ cargo c
    Checking libhandy v0.14.0 (/home/herold/Projects/libhandy-rs/libhandy)
error[E0412]: cannot find type `Widget` in this scope
   --> libhandy/src/auto/deck.rs:534:30
    |
534 |     fn get_child_name<T: IsA<Widget>>(&self, item: &T) -> Option<glib::GString>;
    |                              ^^^^^^ not found in this scope
    |
help: consider importing this struct
    |
5   | use gtk::Widget;
    |

error[E0412]: cannot find type `Widget` in this scope
   --> libhandy/src/auto/deck.rs:536:30
    |
536 |     fn set_child_name<T: IsA<Widget>>(&self, item: &T, name: Option<&str>);
    |                              ^^^^^^ not found in this scope
    |
help: consider importing this struct
    |
5   | use gtk::Widget;
    |

error[E0412]: cannot find type `Widget` in this scope
   --> libhandy/src/auto/deck.rs:799:30
    |
799 |     fn get_child_name<T: IsA<Widget>>(&self, item: &T) -> Option<glib::GString> {
    |                              ^^^^^^ not found in this scope
    |
help: consider importing this struct
    |
5   | use gtk::Widget;
    |

error[E0412]: cannot find type `Widget` in this scope
   --> libhandy/src/auto/deck.rs:814:30
    |
814 |     fn set_child_name<T: IsA<Widget>>(&self, item: &T, name: Option<&str>) {
    |                              ^^^^^^ not found in this scope
    |
help: consider importing this struct
    |
5   | use gtk::Widget;
    |

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0412`.
error: could not compile `libhandy`

To learn more, run the command again with --verbose.
```

--- a/src/codegen/child_properties.rs
+++ b/src/codegen/child_properties.rs
@@ -7,6 +7,7 @@ use crate::{
     chunk::Chunk,
     env::Env,
     library,
+    nameutil::use_gtk_type,
     traits::IntoString,
     writer::{primitives::tabs, ToCode},
 };
@@ -89,7 +90,7 @@ fn declaration(env: &Env, prop: &ChildProperty, is_get: bool) -> String {
         let child_type = RustType::try_new(env, typ).into_string();
         format!("T: IsA<{}>", child_type)
     } else {
-        "T: IsA<Widget>".to_string()
+        format!("T: IsA<{}>", use_gtk_type(env, "Widget"))
     };
     if !is_get && !prop.bounds.is_empty() {
         bounds = format!("{}, {}", prop.bounds, bounds);

